src/Task/AbstractTask.php

```diff
@@ -25,6 +25,9 @@
  */
 abstract class AbstractTask implements TaskInterface
 {
+    public const MIN_PRIORITY = -1000;
+    public const MAX_PRIORITY  = 1000;
+
```

Comment:
If we don't use the const outside of the class, consider using `private` as visibility.

Reply:
yes, I didn't know if it would be useful to use it outside. if it happens, we will be able to use it in public, you are right